bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVReportHandler.java

```diff
@@ -56,6 +59,19 @@
  */
 @NonNullByDefault
 public class OpenUVReportHandler extends BaseThingHandler {
+    private static final DecimalType ALERT_GREEN = DecimalType.ZERO;
+    private static final DecimalType ALERT_YELLOW = new DecimalType(1);
+    private static final DecimalType ALERT_ORANGE = new DecimalType(2);
+    private static final DecimalType ALERT_RED = new DecimalType(3);
+    private static final DecimalType ALERT_PURPLE = new DecimalType(4);
+
+    private static final Map<State, HSBType> ALERT_COLORS = Map.ofEntries(
```

Comment:
Using `Map.of` would make this a bit cleaner, unless of course the generics checker throws a hissy fit.

Reply:
I really like your reviews, it helps me progress. Thanks